diff --git a/autoload/leaderf.vim b/autoload/leaderf.vim
index c7fff9e..635f4b8 100644
--- a/autoload/leaderf.vim
+++ b/autoload/leaderf.vim
@@ -634,6 +634,8 @@ endfunction
 function! leaderf#Quit(manager_id) abort
 exec g:Lf_py "<< EOF"
 import ctypes
+" intvalue = int(vim.eval("a:manager_id"))
+" vim.command(f'echoerr "{intvalue} - type: {type(intvalue)}"')
 manager = ctypes.cast(int(vim.eval("a:manager_id")), ctypes.py_object).value
 if manager.is_ctrl_c == False:
     manager.is_autocmd = True
